subcommands/keys/tuf_rotate_offline_key.go

```diff
@@ -35,6 +35,20 @@ The new signing key is rotated in both CI and production TUF root transactionall
 When you rotate the TUF targets offline signing key:
 - if there are any production targets in your factory, they are re-signed using the new key.
 - if there is an active wave in your factory, the TUF targets rotation is not allowed.`,
+		Example: `
+  # Take ownership of TUF root and targets keys for a new factory, keep them in separate files:
+  fioctl keys tuf rotate-offline-key --role=root \
+    --creds=tuf-root-keys.tar.gz --first-time
+  fioctl keys tuf rotate-offline-key --role=targets \
+    --creds=tuf-root-keys.tar.gz --targets-creds=tuf-targets-keys.tar.gz
+
+  # Rotate offline TUF root key and set the root role expiration to 2 years (the default is 1 year):
+  fioctl keys tuf rotate-offline-key --role=root \
+    --creds=tuf-root-keys.tar.gz --expires-days=730
```

Comment:
I almost objected to the previous commit that allows you to customize these dates for this very reason. Let's just not even let people know they can do bad things.

Reply:
There are two reasons why I made it configurable:
1. This piece of code/logic is a copy-paste from `wave init` where we already allow to customize the expiration (i.e. cross-feature parity).
2. The default expiration period for TUF root is quite controversial: notary suggests 10 years, uptane sets it to 1 year; while garage-sign has no defaults - the user must choose.

Thus far, can you state at least one reason why you think it is "bad thing"?

Reply:
10 years is bad. Ricardo has posted links where someone like NIST has decent guidance. 1 year is a good value. Giving options just takes something a user is scared about and gives them something else to fear and worry about. Then they start thinking "should i use expires-days or expires-at? i wonder why there are two options, do they do the same thing?". Then 30 minutes later they exit the rabbit hole.

Reply:
I agree that 1 year is a sensible value; that's why it is the default. My guts feeling is that customers will ask for this rather sooner than later: some will want it more aggressive like 3-6 years; some will be more happy with 2-3 years. In general I tend to agree on minimalistic command designs (preferably with less than 5 options), but it is a hard game about clarity versus convenience. E.g. I miss some options in `pip install` despite it having 30 options I don't use.

That said, if you insist I will cut this commit; it is fairly easy to add it at any time later.

Reply:
i think we should go for the apple approach here and just say "no, you can't do over 1 year because its not a good idea". and - if someone absolutely insists. there are options:
* the api allows it already
* they can fork fioctl
* we can add this in
Either way - making it painful to do something against recommendations isn't the worst thing

Reply:
Remove this commit from the final force-push